awx/ui_next/src/components/ClipboardCopyButton/ClipboardCopyButton.jsx

```diff
@@ -40,7 +40,13 @@ class ClipboardCopyButton extends React.Component {
   };
 
   render() {
-    const { clickTip, entryDelay, exitDelay, hoverTip } = this.props;
+    const {
+      clickTip,
+      entryDelay,
+      exitDelay,
+      hoverTip,
```

Comment:
can we change the name to copyTip copiedSuccessTip or something like that?  It's currently confusing, because it is the tip content that is changing...there is no difference in the way the tip is triggered and displayed as "click and hover" would have you believe.